### PR TITLE
[1497] Add round settings tab content to round settings page

### LINKED ISSUE #1497: Add Round Settings page > Funding Settings tab
**User Story:**
As a program manager
I want to be able to edit my round details
So that I can make necessary changes before the round starts

**Acceptance Criteria**
GIVEN that I am on my round's details page
WHEN I click on the Round Settings tab 
THEN I see the round settings page
AND the three subtabs (Round Details, Round & Application Period, Funding Settings)
AND the Funding Settings subtab is selected

GIVEN that I am on my round's Round Settings tab
WHEN I am on a subtab
THEN I see the current round settings for that subtab

**Product & Design Links:**
[Figma](https://www.figma.com/file/guY5OKIgM18SNa62OysBoe/%5BWIP%5D-Linear-QF-Flow-v2?node-id=143-8834&t=oaaQsqSd6tVU7JGu-4)
[Loom of UX](https://www.loom.com/share/30521a647d074d49a7bc306809fbe450)

**Tech Details:**

**Open Questions:**

**Notes/Assumptions:**
- This includes writing test coverage for the functionality added in the ticket, if necessary

### PR DESCRIPTION
**Pull request by DanieleSalatti**
##### Description

What it says on the tin

##### Refers/Fixes

Closes #1497 

##### Testing

To be added with write logic

### CODE REVIEW

**Review by thelostone-mc — APPROVED** [review]

File: packages/round-manager/src/features/round/ViewRoundSettings.tsx [hunk 1]
```diff
@@ -0,0 +1,519 @@
...
+        Matching Cap
+      </span>
+      <div className="grid grid-cols-2 grid-rows-1 gap-4 mb-4">
+        <div>
+          <div
+            className={
+              "text-sm leading-5 font-semibold pb-1 flex items-center gap-1 mb-2"
+            }
+          >
+            Do you want a matching cap for projects?
```
Comment (bhargavaparoksham) [inline packages/round-manager/src/features/round/ViewRoundSettings.tsx:360]: If there is no matching cap, the page is failing to load with the following error:

ViewRoundSettings.tsx:366 Uncaught TypeError: Cannot read properties of undefined (reading 'matchingCap')

**Review by bhargavaparoksham — CHANGES_REQUESTED** [review]

**Comment by michellema1208** [comment]
@bhargavaparoksham I just created a round without a matching cap and it looks like the page is loading? Not sure if I did something differently

Screen Shot 2023-04-20 at 9 29 46 AM

cc: @DanieleSalatti @thelostone-mc

**Comment by bhargavaparoksham** [comment]
@michellema1208 I think the round I tested with earlier was an older round which is why it was not working. Rechecked again with a new round. Its working from my end as well.

**Review by bhargavaparoksham — APPROVED** [review]

**Comment by willsputra** [comment]
looking good @DanieleSalatti !

a few ui feedback:
- [ ] for the tabs, could we use the same style as the one in Round Applications? with the blue underline but remove the numbers
image
Round Applications:
image

- [ ] the time says UTC UTC

image

- [ ] the disabled input boxes still kinda look active. could we add a light gray background like we have on Builder?

image
Builder:
image

**Pull request merged**